packages/core/lib/types.ts

```diff
@@ -72,6 +73,9 @@ export interface RenderGraphiQLOptions {
   hybridSubscriptionTransportConfig?: HybridSubscriptionTransportConfig;
 }
 
+type ExecuteFn = (args: ExecutionArgs) => ExecutionResult | Promise<ExecutionResult | AsyncIterable<ExecutionResult>>;
```

Comment:
```suggestion
type ExecuteFn = (args: ExecutionArgs) => ExecutionResult | Promise<ExecutionResult | AsyncGenerator<ExecutionResult>>;
```

Those were typed loosly to prevent TypeScript errors with experimental versions.

Reply:
AsyncIterable should cover AsyncGenerator as well, no?